src/vue/forms/DepositForm.vue

```diff
@@ -90,11 +111,22 @@ export default {
   computed: {
     ...mapGetters([
       vuexTypes.accountId,
+      vuexTypes.account,
+      vuexTypes.wallet,
     ]),
+    balanceId () {
+      return this.account.balances.find(item => {
+        return item.asset.id === this.selectedAsset.code
+      }).id
+    },
   },
   watch: {
     'selectedAsset.code' () {
-      this.disableForm()
+      // Transferred to the disabled state before receiving the address,
+      // in order to avoid a large number of requests
+      if (!this.selectedAsset.isCoinpayments) {
```

Comment:
Please add a task to remove this stupid crutch. Waiting for the task link

Reply:
https://tokend.atlassian.net/secure/RapidBoard.jspa?rapidView=2&projectKey=TDV&modal=detail&selectedIssue=TDV-300